src/optim/FishLeg/fishleg.py

```diff
@@ -179,6 +228,7 @@ def __init__(
         self.beta = beta
         self.pre_aux_training = pre_aux_training
         self.step_t = 0
+        self.store_g = True
```

Comment:
Is this changeable? If so, it belongs in the init with True as a default. If not then probably not needed here.

Reply:
Yes, changeable and initialised as True.